portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,37 +33,21 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-
-  private ArrayList<String> shows;
-
+ 
   @Override
-  public void init(){
-      shows = new ArrayList<String>();
-      shows.add("The Good Doctor");
-      shows.add("Grey\'s Anatomy");
-      shows.add("House");
-      shows.add("Code Black");
-  }
-
-  @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    //String show = shows.get((int) (Math.random() * shows.size()));
-    //response.setContentType("text/html;");
-    //response.getWriter().println(show);
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+      //shows = new ArrayList<String>();
+      String text = request.getParameter("text-input");
+      long timestamp = System.currentTimeMillis();
 
+      Entity taskEntity = new Entity("Comments");
```

Comment:
Any reason you are calling these "Comments" instead of "Shows"?

Reply:
I called them comments because the task for the week was to implement a way for someone to post a comment on the page. Shows was from list I made earlier containing T.V. shows and the name stuck around in my code.

Reply:
Ah.  I had assumed that it had started as comments and was being converted to shows.  In that case, would it make more sense to call your new class "Comment" instead of "Show" then?